build/app.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cbroglie/mustache"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	sigsYamlFile   = "sigs.yaml"
+	templateDir    = "build"
+	indexTemplate  = fmt.Sprintf("%s/sig_index.mustache", templateDir)
+	listTemplate   = fmt.Sprintf("%s/sig_list.mustache", templateDir)
+	footerTemplate = fmt.Sprintf("%s/footer.mustache", templateDir)
+	sigListOutput  = "sig-list.md"
+	sigIndexOutput = "README.md"
+)
+
+type Lead struct {
+	Name    string
+	Company string
+	GitHub  string
+}
+
+type Meeting struct {
+	Day       string
+	UTC       string
+	PST       string
+	Frequency string
+}
+
+type Contact struct {
+	Slack       string
+	MailingList string `yaml:"mailing_list"`
+	GitHubTeam  string `yaml:"github_team"`
```

Comment:
Github team has gotten more complicated.  See https://github.com/kubernetes/community/blob/master/sig-creation-procedure.md.  Not every SIG has done this dance yet though.

Idea: have a bool ("FullGitHubTeams") and if that is try then we can expand out the team according to the link above.  If false, just don't list any github teams.